yarn.lock

```diff
@@ -2527,7 +2527,7 @@ eslint-plugin-flowtype@^2.30.0:
     lodash "^4.15.0"
 
 "eslint-plugin-guardian-frontend@file:dev/eslint-plugin-guardian-frontend":
-  version "3.0.0"
+  version "4.0.0"
```

Comment:
Irrelevant for now as this change has already found its way into master.

It will be worth considering in future that if we add lint rules to our plugin, they won't automatically get added to `yarn.lock`.